kpops/component_handlers/helm_wrapper/helm.py

```diff
@@ -68,7 +68,13 @@ def add_repo(
             else:
                 raise e
 
-        self.__execute(["helm", "repo", "update"])
+        version = self.get_version()
+        _, minor, _ = map(int, version.split("."))
+
+        if minor > 7:
```

Comment:
shouldn't we also check that major >= 3?

Reply:
I was not really sure about that. But I think it is not a bad idea. Maybe I can push the check of the major a bit further up in the method.

Reply:
Do major versions support backwards compatible syntax, i.e. could major > 3 introduce a breaking change?

Reply:
good point! I would also prefer pinning the major version to 3. Once Helm 4 becomes a thing we get a chance to confirm it is compatible.